### DISCUSSION TITLE
203에러

### DISCUSSION THREAD

**Discussion by RoRoGUMI** [storycraft/node-kakao#525] [Q&A] [answered]
203 에러로 유지가 안되네요 로그에서는 이렇게 나오는데 203에러 어떻게 해결할 수 있을까요?

(node:1) UnhandledPromiseRejectionWarning: Error: Client login failed with status: -203
(Use `node --trace-warnings ...` to show where the warning was created)
(node:1) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)      
(node:1) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

**Comment by secneuqes** [comment]
이 문제로 저도 유지가 안되고 있습니다... 해결책을 알려주실 분을 찾고 있습니다

**Reply by MoonJR** [reply]
4.3.1 버전으로 올리시면 발생 안하실것입니다. @kingsparkdev @fxcorea12

**Comment by storycraft** [comment, marked as answer]
node-kakao 버전이 최신버전 인가요?
